src/components/Button/Button.test.tsx

```diff
@@ -9,4 +9,9 @@ describe('Button component', () => {
     const button = screen.getByRole('button');
     expect(button).toBeInTheDocument();
   });
+
+  it('matches snapshot', () => {
+    const { asFragment } = contextRender(<Button>Hello!</Button>);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
I wasn't sure if I should use `container`, `baseElement` or `asFragment()`.

See https://testing-library.com/docs/react-testing-library/api/#container-1 for docs on those three. Not sure if there's a recommended property to use for saving the snapshot.